src/FacesRatingBar/constants.js

```diff
@@ -0,0 +1,40 @@
+export const dataHooks = {
+  face: 'face',
+  disapointedTooltip: 'disapointedTooltip',
+  frowningTooltip: 'frowningTooltip',
+  neutralTooltip: 'neutralTooltip',
+  smilingTooltip: 'smilingTooltip',
+  griningTooltip: 'griningTooltip',
+};
+
+export const tooltipDataHookMap = {
+  1: 'disapointedTooltip',
+  2: 'frowningTooltip',
+  3: 'neutralTooltip',
+  4: 'smilingTooltip',
+  5: 'griningTooltip',
+};
+
+export const facesRatingBarSizes = {
+  medium: 'medium',
+};
+
+export const faceIndexes = {
+  1: 1,
+  2: 2,
+  3: 3,
+  4: 4,
+  5: 5,
+};
+
+export const faceIconTypeMap = {
+  1: 'disapointed',
+  2: 'frowning',
+  3: 'neutral',
+  4: 'smiling',
+  5: 'grining',
+};
+
```

Comment:
You can have one data object:

```suggestion
export const facesData = {
  1: { name: 'disapointed', dataHook: 'face-disapointed' },
  2: .....,
  3: .....,
  4: .....,
  5: .....,
};
```